### DISCUSSION TITLE
Pre populate workspace on install

### DISCUSSION THREAD

**Discussion by lukeantbits** [RocketChat/Rocket.Chat.ReactNative#5331] [Q&A]
I'm looking to build a white label version of the App, I have it up and running on my simulator. How would I go about bypassing the enter workspace view? I'd like to simplify the user experience and have the workspace already populated in the database on install, with users proceeding straight to the login view

**Comment by dnlsilva** [comment]
Use the single server branch:
https://github.com/RocketChat/Rocket.Chat.ReactNative/tree/single-server

https://developer.rocket.chat/open-source-projects/mobile-app/mobile-app-white-labelling
